### ISSUE TITLE
MUC/group chat: Message double, shown from sender AND recipient

### ISSUE THREAD

**Issue by Echolon** [anurodhp/Monal#216]
Hello,

first take a look at this screenshot

<img src="https://user-images.githubusercontent.com/29168011/53682379-37a93180-3cf5-11e9-9657-e1867df359b9.jpg" width="350">

It seems that if a Monal user sends a message in a group chat, the message will be shown normally but also as double from a 'test' recipient. This double is only shown for the Monal user.

Hope this description helps. The user in this case uses a prosody server (jabber.de). Happened after I invited him to a private group chat. Message is not encrypted.

**Comment by janteau** [comment]
I also experience this issue.  Messages sent are displayed as also coming from a 'test' user.  No encryption is used - OMEMO does not seem to work at all in MUC.

**Comment by decentral1se** [comment]
Yes, badly affected by this. Group chat is unusable unfortunately.

Lovely app otherwise :sun_with_face:

**Comment by Echolon** [comment]
@janteau  @decentral1se 

Hi,

do you still experience this?

**Comment by janteau** [comment]
I don't believe this particular issue still exists.  Though I tried retesting this last night in 4.2.1 and can't get Monal to join a Private MUC.  A notification is received but Monal never prompts to actually join the room.  I have tried manually adding the group address but that wasn't working either

**Comment by licaon-kter** [comment]
@janteau OMEMO in groups is a known missing thing, was your test group unencrypted?

**Comment by janteau** [comment]
@licaon-kter Yes, I never even get inside the group.  Manually adding the group, messages sent are not seen inside the group (in Coversations unencrypted) and sending unencrypted messages from Conversations are not seen either.

**Comment by janteau** [comment]
I just retested this.  I did get a Join message prompt, but I had the app up and open when I sent the invite from Conversations.

I can now receive messages from Conversations, but the messages sent from Monal do not appear in Conversations.
<img src="https://user-images.githubusercontent.com/32575346/71785361-8c44c980-2ff6-11ea-8e84-0d614a8aa800.jpg" width="300" />
<img src="https://user-images.githubusercontent.com/32575346/71785363-9070e700-2ff6-11ea-9bd4-15492d2ba669.png" width="300" />
